packages/headless/src/controllers/commerce/facets/core/numeric/headless-commerce-numeric-facet.ts

```diff
@@ -0,0 +1,56 @@
+import {CommerceEngine} from '../../../../../app/commerce-engine/commerce-engine';
+import {
+  toggleExcludeNumericFacetValue,
+  toggleSelectNumericFacetValue,
+} from '../../../../../features/facets/range-facets/numeric-facet-set/numeric-facet-actions';
+import {
+  CoreCommerceFacet,
+  CoreCommerceFacetOptions,
+  NumericFacetValue,
+  NumericRangeRequest,
+  buildCoreCommerceFacet,
+} from '../headless-core-commerce-facet';
+
+export type CommerceNumericFacetOptions = Omit<
+  CoreCommerceFacetOptions,
+  'toggleSelectActionCreator' | 'toggleExcludeActionCreator'
+>;
+
+/**
+ * The `CommerceNumericFacet` controller offers a high-level programming interface for implementing numeric commerce
+ * facet UI component.
+ */
+export interface CommerceNumericFacet
+  extends CoreCommerceFacet<NumericRangeRequest, NumericFacetValue> {}
+
+/**
+ * @internal
+ *
+ * **Important:** This initializer is meant for internal use by headless only.
+ * As an implementer, you must not import or use this initializer directly in your code.
+ * You will instead interact with `CommerceNumericFacet` controller instances through the state of a `FacetGenerator`
+ * controller.
+ *
+ * @param engine - The headless commerce engine.
+ * @param options - The `CommerceNumericFacet` options used internally.
+ * @returns A `CommerceNumericFacet` controller instance.
+ */
+export function buildCommerceNumericFacet(
+  engine: CommerceEngine,
+  options: CommerceNumericFacetOptions
+): CommerceNumericFacet {
+  const coreController = buildCoreCommerceFacet<
+    NumericRangeRequest,
+    NumericFacetValue
+  >(engine, {
+    options: {
+      ...options,
+      toggleSelectActionCreator: toggleSelectNumericFacetValue,
+      toggleExcludeActionCreator: toggleExcludeNumericFacetValue,
+    },
+  });
+
+  return {
+    ...coreController,
+  };
```

Comment:
No need for the intermediary `const`. We can could return the controller right away:
```suggestion
  return buildCoreCommerceFacet<
    NumericRangeRequest,
    NumericFacetValue
  >(engine, {
    options: {
      ...options,
      toggleSelectActionCreator: toggleSelectNumericFacetValue,
      toggleExcludeActionCreator: toggleExcludeNumericFacetValue,
    },
  });
```
Same comment applies for the `buildCommerceRegularFacet`!